### DISCUSSION TITLE
Game Gear MiSTer core didn't see any roms because symlink wasn't there - I had to add it manually

### DISCUSSION THREAD

**Discussion by zezretro** [retronas/discussions#122] [General]
Does anyone else have an issue with the GameGear symlink not appearing in /data/retronas/mister/games?

In my GG mister core, I wasn't seeing any roms. Then I found the sym link wasn't there.

I tried running the core roms directory option in the retronas admin and that did not recreate it.

Eventually I added it back myself manually and now everything is fine again

ln -s /data/retronas/roms/sega/gamegear /data/retronas/mister/games/GameGear

**Comment by sairuk** [comment]
No, i've just tried to replicate this but The GameGear folder was created successfully with the installer

The core rom directory option creates the base retronas directory structure not the system (e.g. mister) symlinks, to build out the system symlinks you'll need to rerun the installer. With newer mister installations we install an auto runner/updater for mister directory builds

**Reply by zezretro** [reply]
Coolio, I am glad it was just me
